docs/5.0/config-reference.md

```diff
@@ -323,8 +323,11 @@ proxy_service:
 
     # TLS certificate for the HTTPS connection. Configuring these properly is
     # critical for Teleport security.
-    https_key_file: /var/lib/teleport/webproxy_key.pem
-    https_cert_file: /var/lib/teleport/webproxy_cert.pem
+    https_keypairs:
+    - key_file: /var/lib/teleport/webproxy_key.pem
+    - cert_file: /var/lib/teleport/webproxy_cert.pem
+    - key_file: /etc/letsencrypt/live/*.teleport.example.com/privkey.pem
+    - cert_file: /etc/letsencrypt/live/*.teleport.example.com/fullchain.pem
```

Comment:
We have to be really careful with this - each entry is a "pair" in the YAML list so the config should only have one `-` per key/cert combination. Teleport will fail to start if you add a `-` to every line.

```suggestion
    https_keypairs:
    - key_file: /var/lib/teleport/webproxy_key.pem
      cert_file: /var/lib/teleport/webproxy_cert.pem
    - key_file: /etc/letsencrypt/live/*.teleport.example.com/privkey.pem
      cert_file: /etc/letsencrypt/live/*.teleport.example.com/fullchain.pem
```